advanced/lighttpd.conf.debian

```diff
@@ -36,6 +36,11 @@ server.port                 = 80
 accesslog.filename          = "/var/log/lighttpd/access.log"
 accesslog.format            = "%{%s}t|%V|%r|%s|%b"
 
+# Allow streaming response
+# reference: https://redmine.lighttpd.net/projects/lighttpd/wiki/Server_stream-response-bodyDetails
+server.stream-response-body = 1
+ssl.read-ahead              = "disable"
```

Comment:
```suggestion
ssl.read-ahead              = "disable"
```
That is the default value and the config requires `mod_openssl` to be enabled. Can we remove it and see if that helps the testing situation?